cmd/ygg/main.go

```diff
@@ -154,8 +154,58 @@ func main() {
 		{
 			Name:  "status",
 			Usage: "reports status of connection and activation",
+			Flags: []cli.Flag{
+				// TODO: load this value from config ? (or have a default)
+				&cli.StringSliceFlag{
+					Name: "broker",
+				},
+			},
 			Action: func(c *cli.Context) error {
-				return fmt.Errorf("not implemented")
+				hostname, err := os.Hostname()
+				if err != nil {
+					// default for messaging if hostname can't be read
+					hostname = "host"
+				}
+				fmt.Printf("Connection status for %s:\n\n", hostname)
+
+				// get RHSM registration status
+				uuid, err := getConsumerUUID()
+				if err != nil {
+					fmt.Printf("⛔️ ERROR: Unable to check Red Hat Subscription Manager status: %s\n\n", err)
+				} else {
+
+					if uuid != "" {
+						fmt.Println("✅ Connected to Red Hat Subscription Manager (RHSM).")
+					} else {
+						fmt.Println("❌ Not connected to Red Hat Subscription Manager (RHSM).")
+					}
+				}
+
+				// get broker connection status
+				errMsg := "⛔️ ERROR: Unable to connect to Cloud Client service: %s\n\n"
```

Comment:
Until we have a D-Bus API for the client program to communicate with the service over, I think I'd prefer to keep this just reporting whether or not the `yggd.service` is running. This implementation bleeds a lot of daemon features into the client.